### PR TITLE
Show live revisions count instead of total revisions

### PR DESCRIPTION
**Pull request by QEDK**
Tiny fix

### CODE REVIEW

File: templates/editCounter/general_stats.wikitext.twig [hunk 1]
```diff
@@ -30,7 +30,7 @@
 {% endif %}
 |-
 | {{ msg('is-admin') }}
-| {% if user.isAdmin(project) %}1{% else %}0{% endif %}
+| {% if user.isAdmin(project) %}Yes{% else %}No{% endif %}
```
Comment (MusikAnimal) [inline templates/editCounter/general_stats.wikitext.twig]: This needs to be localized. I can show you how to do this if you'd like, but regardless I think this change should be saved for a separate PR. We could even show images for a green tick and red cross, like we do in the UI (but putting "yes" / "no" as the `alt` text, for accessibility).

**Review by MusikAnimal — CHANGES_REQUESTED** [review]
Thank you for the fix!

If we can move the yes/no change to a separate PR then I will merge :)

**Comment by QEDK** [comment]
@MusikAnimal done.

**Comment by MusikAnimal** [comment]
Thank you!

**Pull request merged**